src/NuGet.Clients/NuGet.Console/ConsoleInitializer.cs

```diff
@@ -45,18 +37,14 @@ private static Task<Action> GetInitializeTask()
                 // which is very rare.
             }
 
-            var initializer = componentModel.GetService<IHostInitializer>();
-            return Task.Factory.StartNew(state =>
```

Comment:
There is potential change in behavior here. Content of delegate passed to StartNew was executed on background thread, and now is executing on whatever thread executes GetInitializeTaskAsync. Could be the same, but please verify.

Reply:
Just in case I reverted some part of code, please check.